lib/core/src/lib/notifications/components/notification-history.component.ts

```diff
@@ -145,4 +156,8 @@ export class NotificationHistoryComponent implements OnDestroy, OnInit, AfterVie
             this.trigger.closeMenu();
         }
     }
+
+    isbadgeVisible(): boolean {
```

Comment:
typo, should be isBadgeVisible (not isbadgeVisible)

Reply:
sorted now